### DISCUSSION TITLE
Only one Bluetooth device for 'speak reading' ?

### DISCUSSION THREAD

**Discussion by bjarkih1977** [NightscoutFoundation/xDrip#2548] [General]
I'm having a bit of trouble with bluetooth devices and vehicle mode/speak readings.  I love having the function of xDrip reading the values to me when connected to bluetooth audio while e.g driving.  Problem is that I use another BT device at work and xDrip doesn't seem to store the info on both so at the start of work I have to pair xDrip to my work BT audio and vice versa after work when starting up the car.  Any suggestions?  Other apps on the phone are fine switching between BT devices.

**Comment by Navid200** [comment]
Just for reference, would you please give an example of an app that can automatically switch from one Bluetooth device to another with no need to pair every time?

**Reply by bjarkih1977** [reply]
Spotify.  Pause it when quiting work, turn of the BT device I use at work.  Go to car, turn car on and resume playing music.

**Comment by tolot27** [comment]
That should be simple to fix. @Navid200 I would recommend to create an issue from this discussion and postponed it for later implementation.

**Comment by Navid200** [comment]
@bjarkih1977 Please convert to issue by clicking on "Create issue from discussions" in the right pane.
Thanks
